### DISCUSSION TITLE
How to create custom multi-level serializer?

### DISCUSSION THREAD

**Discussion by skasidej** [encode/django-rest-framework#7891] [Question & Answer]
Let's say I want to create a custom serializer for a complex multi-level json request. How am I suppose to do it in the correct way?

For example this request body are using the same serializer for address details, but how should I create serializer for the locale schema part?

`
address: {
  **address_detail,
  locale: {
    th: { **address_detail }
  }
}
`

**Comment by auvipy** [comment]
you can check this package https://github.com/beda-software/drf-writable-nested
